src/solarwinds-node.js

```diff
@@ -28,11 +29,28 @@ program
 program
   .command('list')
   .alias('ls')
+  .option('--filter <filter>', 'Filter by key, for example Name, Hostname, IP or ID')

```

Comment:
change to `.option('--filter <value>', 'Filter output based on conditions provided')`